src/views/AdminEventLog.vue

```diff
@@ -15,13 +30,13 @@
                 item-key="key"
                 loading-text="Loading events"
                 :loading="loadingStatus"
-                :search="search"
+                :search="filterEvents"
         >
             <template v-slot:expanded-item="{ headers, item }">
                 <td :colspan="headers.length"><pre>{{item.representation | pretty}}</pre></td>
             </template>
         </v-data-table>
-        <button @click="getAdminEvents">Refresh</button>
+        <button @click="getAllEvents">Refresh</button>
```

Comment:
This "Refresh" button will probably just be removed, actually. Everything can be done with the "Search" button. It was just useful when I first started developing this.